static/js/place/menu.tsx

```diff
@@ -45,6 +45,14 @@ class MenuCategory extends React.Component<MenuCategoryPropsType> {
           href={hrefString}
           className={`nav-link ${selectCategory === category ? "active" : ""}`}
           text={this.props.categoryDisplayStr}
+          handleClick={() =>
+            window &&
+            window.gtag &&
+            window.gtag("event", "place_category_click", {
+              place_category_click: category,
+              place_category_click_source: "menu",
```

Comment:
nit: can we call this `sidebar` to be more specific (we ought to rename the component too, but that's for later).

Reply:
Done